discord/message.py

```diff
@@ -1525,9 +1533,10 @@ async def create_thread(self, *, name: str, auto_archive_duration: ThreadArchive
             self.channel.id,
             self.id,
             name=name,
-            auto_archive_duration=auto_archive_duration or self.channel.default_auto_archive_duration,
+            # if the channel isn't a TextChannel, this will raise AttributeError 
+            auto_archive_duration=auto_archive_duration or self.channel.default_auto_archive_duration,  # type: ignore
```

Comment:
I fixed this differently locally, can you remove it?

Reply:
Done.